src/measure_theory/measurable_space.lean

```diff
@@ -860,8 +871,51 @@ end sum
 instance {α} {β : α → Type*} [m : Πa, measurable_space (β a)] : measurable_space (sigma β) :=
 ⨅a, (m a).map (sigma.mk a)
 
+section prop
+variables {p : α → Prop}
+
+variables [measurable_space α]
+
+@[simp] lemma measurable_set_set_of : measurable_set {a | p a} ↔ measurable p :=
+begin
+  refine ⟨λ h s _, _, λ h, by simpa using h (measurable_set_singleton true)⟩,
+  by_cases hs₁ : true ∈ s; by_cases hs₀ : false ∈ s,
```

Comment:
I bet you can golf using `measurable_to_countable` or `measurable_to_countable'`.